MVVMIOS/Models/Response.swift

```diff
@@ -7,6 +7,44 @@
 
 import Foundation
 
-struct Response: Codable {
+struct Response: Codable, Equatable {
     let name: String
 }
+
+struct News<T: Codable>: Codable {
+    let data: T
+}
+
+struct Pagination: Codable {
+
+}
+
+struct NewsData: Codable {
+    let id: UUID = UUID()
```

Comment:
I don't think defaults are actually used in Codable structs are they? I think this will work the same using the below.
```suggestion
    let id: UUID? 
```